stories/ClaimDetails.stories.tsx

```diff
@@ -7,6 +7,38 @@ export default {
   title: 'Component/Page Section/Claim Details',
   component: ClaimDetailsComponent,
   decorators: [withNextRouter],
+  argTypes: {
+    programType: {
+      control: {
+        type: 'text',
+      },
+    },
+    benefitYear: {
+      control: {
+        type: 'text',
+      },
+    },
+    claimBalance: {
+      control: {
+        type: 'text',
+      },
+    },
+    weeklyBenefitAmount: {
+      control: {
+        type: 'text',
+      },
+    },
+    lastPaymentIssued: {
+      control: {
+        type: 'text',
+      },
+    },
+    extensionType: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
 } as Meta
 
 const Template: Story<ClaimDetailsProps> = (args) => <ClaimDetailsComponent {...args} />
```

Comment:
This is definitely not a result of your PR, but while reviewing I noticed that the `lastPaymentIssued` story arg should probably be `'$120.00 on 4/29/21'` or something like that.